portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,62 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function randomizeImage() {
+  // The images directory contains 13 images, so generate a random index between
+  // 1 and 13.
+  const imageIndex = Math.floor(Math.random() * 24) + 1;
+  const imgUrl = 'images/jfew-' + imageIndex + '.jpg';
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const imgElement = document.createElement('img');
+  imgElement.src = imgUrl;
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const imageContainer = document.getElementById('random-image-container');
+  // Remove the previous image.
+  imageContainer.innerHTML = '';
+  imageContainer.appendChild(imgElement);
+}
+
+function openIntromodal() {
+  const button = document.getElementById("Intro");
+  const modal = document.getElementById("myIntroModal");
+  modal.style.display = "block";
+}
+
+function openPicturemodal() {
+  const button = document.getElementById("Pictures");
+  const modal = document.getElementById("myPictureModal");
+  modal.style.display = "block";
+}
+
+function openConnectmodal() {
+  const button = document.getElementById("Connect");
+  const modal = document.getElementById("myConnectModal");
+  modal.style.display = "block";
+}
+
+function closeIntromodal(){
+  const span = document.getElementsByClassName("close")[0];
+  const modal = document.getElementById("myIntroModal");
+  modal.style.display = "none";
+}
+
+function closePicturemodal(){
+  const span = document.getElementsByClassName("close")[0];
+  const modal = document.getElementById("myPictureModal");
+  modal.style.display = "none";
+}
+
+function closeConnectmodal(){
+  const span = document.getElementsByClassName("close")[0];
+  const modal = document.getElementById("myConnectModal");
+  modal.style.display = "none";
+}
+
+window.onclick = function(event) {
+  const modal = document.getElementById("myIntroModal");
+  const modal2 = document.getElementById("myPictureModal");
+  const modal3 = document.getElementById("myConnectModal");
+  if (event.target == modal) modal.style.display = "none";
```

Comment:
I think you should be using === instead of == because it's safer

https://google.github.io/styleguide/jsguide.html#features-equality-checks